src/js/modules/rpc/server.ts

```diff
@@ -224,6 +225,9 @@ export class ProcessBatchServer extends SupervisorServer {
       );
       return undefined;
     }
+    if (!errors.validateWasmAttributes(handle, id, this.logger)) {
+      return [undefined, errors.validateLoadScriptError(null, id, script)];
```

Comment:
shouldn't this be in a try catch?

Reply:
No, those operation doesn't have side effects